extensions/amp-story/1.0/amp-story-reaction-poll-binary.css

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+.i-amphtml-story-reaction-poll-binary-container {
+    display: flex !important;
+    align-items: stretch !important;
+    justify-content: center !important;
+    height: 90px !important;
+    max-width: 296px !important;
+    border-radius: 16px !important;
+    box-shadow: 0px 2px 4px #000000aa !important;
+    color: var(--reaction-main-color) !important;
+    background: var(--reaction-theme-background) !important;
+    overflow: hidden !important;
+
+    --post-select-scale: 0.5 !important;
+    --post-select-scale-variable: 1 !important;
+}
+
+.i-amphtml-story-reaction-poll-binary-container:not(.i-amphtml-story-reaction-post-selection) {
+    cursor: pointer !important;
+}
+
+[dir="rtl"] .i-amphtml-story-reaction-poll-binary-container {
+    flex-direction: row-reverse !important;
```

Comment:
Actually that makes more sense, not sure why I was trying to re-reverse it, I think it was to keep the black panther example on the same side, but using the default reversed makes more sense, since publishers will be writing and making the stories in RTL layouts from the start.